Makefile

```diff
@@ -26,15 +26,15 @@ database: test-env vendor
 	mysqldump -uroot cfp_test > tests/dump.sql
 
 infection: vendor database
-	vendor/bin/infection --test-framework-options="--printer PHPUnit\\\TextUI\\\ResultPrinter"
+	php -d zend_extension=xdebug.so vendor/bin/infection
 
 integration: test-env vendor database cache
 	vendor/bin/phpunit --testsuite integration
 
 stan: vendor
-	vendor/bin/phpstan analyse --level=2 src tests
+	vendor/bin/phpstan analyse
 
-test: auto-review integration unit
+test: auto-review integration unit stan
```

Comment:
What do you think about adding `stan` to the `it` target instead?

Then it will be run as well when we run

```
$ make it
```

or 

```
$ make
```

Reply:
done